src/providers/VoiceFocusProvider/index.tsx

```diff
@@ -67,8 +67,11 @@ const VoiceFocusProvider: React.FC<Props> = ({
   });
 
   const addVoiceFocus = async (device: Device): Promise<Device | VoiceFocusTransformDevice> => {
+    console.info('Chose a new audio input device', device);
```

Comment:
nit: Chose -> Choose ?

I used Chose, to specify that the choosing is already finished :)

Reply:
Yes, I think that the device has already been chose when `addVoiceFocus` is triggered, so use the past tense

Reply:
Then, should we say?
console.log('Add voice focus to following audio input device', device);